cluster/gce/gci/configure-helper.sh

```diff
@@ -1027,7 +1046,12 @@ function start-kube-proxy {
   fi
   local container_env=""
   if [[ -n "${ENABLE_CACHE_MUTATION_DETECTOR:-}" ]]; then
-    container_env="env:\n    - name: KUBE_CACHE_MUTATION_DETECTOR\n    value: \"${ENABLE_CACHE_MUTATION_DETECTOR}\""
+    # Daemonset manifest needs a different indent.
```

Comment:
Why kube-proxy related change causes the update for cache_mutation_detector?

Reply:
> This is very error-prone, and hard to maintain in a long run.

Agreed, I removed such logic and make it consistent among both daemonset and static pod manifests.